packages/type-utils/src/typeFlagUtils.ts

```diff
@@ -1,31 +1,42 @@
 import { unionTypeParts } from 'tsutils';
 import * as ts from 'typescript';
 
+const ANY_OR_UNKNOWN = ts.TypeFlags.Any | ts.TypeFlags.Unknown;
+
 /**
- * Gets all of the type flags in a type, iterating through unions automatically
+ * Gets all of the type flags in a type, iterating through unions automatically.
  */
-export function getTypeFlags(type: ts.Type): ts.TypeFlags {
-  let flags: ts.TypeFlags = 0;
+export function getTypeFlags(type: ts.Type): number | ts.TypeFlags {
+  let flags = 0;
```

Comment:
What's actually happening is we're assuming that `flags`, despite starting off as an invalid value for `ts.TypeFlags`, will eventually become one. I think a type assertion is correct here.

```suggestion
export function getTypeFlags(type: ts.Type): ts.TypeFlags {
  let flags = 0 as ts.TypeFlags;
```